### PR TITLE
✨ shortcode carousel

### PR DESCRIPTION
**Pull request by nunocoracao**

### CODE REVIEW

**Comment by nunocoracao** [comment]
@mirceanton new PR with a separate branch for collab

**Comment by nunocoracao** [comment]
@mirceanton we'll also work on this one... needs rework

**Comment by mirceanton** [comment]
@nunocoracao definitely! I'll try to get to it tomorrow. I'll try to do this one though. Your last PR gave me enough of an example, I believe, to fix this one myself. I'll let you know if I run into any issues but hopefully it'll be fine

**Comment by nunocoracao** [comment]
@mirceanton sorry already on it... :D

**Comment by nunocoracao** [comment]
wIll just remove the dependency for this one. We can replicate the same thing using tailwind no need to add a dependency for this. I will push my code today and you can pick it up / review.

**Comment by mirceanton** [comment]
You're too fast for me ☺️

**Comment by nunocoracao** [comment]
The preview links for each PR should also be fixed from another PR I merged today. @mirceanton you'll be able to see the results in the preview link

File: layouts/shortcodes/sliderOLD.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      el: '.swiper-pagination{{ $id }}',
+      clickable: true,
+    },
+  
+    navigation: {
+      nextEl: '.swiper-button-next{{ $id }}',
+      prevEl: '.swiper-button-prev{{ $id }}',
+    },
+  });
+</script>
```
Comment (mirceanton) [inline layouts/shortcodes/sliderOLD.html:46]: I believe this file should be removed now. Right?

File: exampleSite/content/docs/shortcodes/index.md [hunk 2]
```diff
@@ -90,6 +90,26 @@ Call to action
...
 {{< /button >}}
 
+## Carousel
+
+`carousel` is used to showcase multiple images in an interactive and visually appealing way. This allows a user to slide through multiple images while only taking up the vertical space of a single one. All images are displayed using the full width of the parent component and using a ratio of `16:9`.
+
+<!-- prettier-ignore-start -->
+| Parameter | Description                                               |
+| --------- | --------------------------------------------------------- |
+| `images`  | This shortcode receives how many image paths as you want. |
```
Comment (mirceanton) [inline exampleSite/content/docs/shortcodes/index.md:100]: I am not so sure about this table as it kinda suggests that there is a named parameter called `images`. I think the examples should suffice.

File: layouts/shortcodes/carousel.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<div id="carouselExampleCaptions" class="carousel slide relative" data-bs-ride="carousel">
```
Comment (mirceanton) [inline layouts/shortcodes/carousel.html:1]: Here, the ID should either be dynamically generated or user provided. If we leave it like this, we cannot have multiple carousels in the same page, or the controls from either one will just act on the first.

**Comment by mirceanton** [comment]
@nunocoracao See #467 as it fixes all of these comments

File: layouts/shortcodes/sliderOLD.html (hunk 1 above)
Comment (nunocoracao) [reply layouts/shortcodes/sliderOLD.html]: yes thanks

File: layouts/shortcodes/carousel.html (hunk 3 above)
Comment (nunocoracao) [reply layouts/shortcodes/carousel.html]: nice one

File: exampleSite/content/docs/shortcodes/index.md (hunk 2 above)
Comment (nunocoracao) [reply exampleSite/content/docs/shortcodes/index.md]: Saw your changes thanks

**Comment by nunocoracao** [comment]
merging, thanks again @mirceanton

**Pull request merged**